src/test/scala/chiselTests/Vec.scala

```diff
@@ -517,4 +517,19 @@ class VecSpec extends ChiselPropSpec with Utils {
   property("reduceTree should preserve input/output type") {
     assertTesterPasses { new ReduceTreeTester() }
   }
+
+  property("Vecs of empty Bundles should work") {
```

Comment:
can we test a Vec of Record with empty `elements` as well? Since it's technically not a Bundle

Reply:
Sure

Reply:
Done.